### ISSUE TITLE
Different behaviour on linux and windows

### ISSUE THREAD

**Issue by agat-asensus** [CastXML/CastXML#247]
What extra args can I give castXML to use the cpp file as it's running on windows, even if it's running on linux?

**Comment by bradking** [comment]
Whether this is possible, and how to do it, depends on what you want to accomplish.  Can you be more specific?

**Comment by agat-asensus** [comment]
yes, I use Pycstruct library, parse_file function to parse a cpp file that defines a nested structure, to read it's content from a binary file. The problem is the padding of the structured depends on the cpp compiler, that changes on different OS. The file was written on a Windows machine but we try to read it on a Linux machine, where the cpp padding is different. I need it to behave the same (an option that says run like you are on Windows). Thanks

**Comment by bradking** [comment]
CastXML is based on LLVM/Clang, which supports a `--target=` flag to specify the compilation target ABI.  Try adding `--target=x86_64-pc-windows-msvc` to your castxml command line.
